services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/OpenHouseDatabasesApiValidator.java

```diff
@@ -44,6 +44,25 @@ public void validateGetAclPolicies(String databaseId) {
     validateDatabaseId(databaseId);
   }
 
+  @Override
+  public void validateGetAllDatabases(int page, int size, String sortBy) {
+    List<String> validationFailures = new ArrayList<>();
+    if (page < 0) {
+      validationFailures.add(String.format("page : provided %s, cannot be negative", page));
+    }
+    if (size <= 0) {
+      validationFailures.add(String.format("size : provided %s, must be greater than 0", size));
+    }
+    if (sortBy != null && (sortBy.contains(",") || sortBy.contains(":"))) {
+      validationFailures.add(
+          String.format(
+              "sortBy : provided %s, does not support multiple sort fields or directions", sortBy));
+    }
+    if (!validationFailures.isEmpty()) {
+      throw new RequestValidationFailureException(validationFailures);
```

Comment:
Looks like this validation is pretty much common for all the methods. I think we can extract a common validator.

Reply:
Yes.